tests/data/load_test_data.sh

```diff
@@ -0,0 +1 @@
+./dataload/import_to_elasticsearch.py ./dataload/test_data/a002400000KeYdsAAF-currency.json ./dataload/test_data/a002400000OiDBQAA3.json ./dataload/test_data/awefw001p00000zgyHZAAY.json  --clean 
```

Comment:
Running this command (or the script) gives FileNotFound error
```
FileNotFoundError: [Errno 2] No such file or directory: './dataload/test_data/a002400000KeYdsAAF-currency.json'
```
I'm guessing since they've all be moved